### DISCUSSION TITLE
Index.md changed and merged, but changes not reflected on website

### DISCUSSION THREAD

**Discussion by rebullm** [greenelab/lab-website-template#193] [General] [answered]
Hi! 
I deleted the REBULLM's website line from index (I actually tried to comment it out first) and while currently my index.md file doesn't have this header, my website does. 

https://rebullm.github.io/Bennett-Lab-website-v1.1/

Do you know why or how I can fix it? 
Thanks, 
Meg

**Comment by vincerubinetti** [comment, marked as answer]
I do not see the "rebullm's Website" heading when I go to that link:

<img width="941" alt="Screenshot 2023-04-27 at 3 35 32 PM" src="https://user-images.githubusercontent.com/8326331/234972549-271960a6-b690-43e1-8bcf-1f7758831a37.png">

Have you tried hard-refreshing the page? Sometimes hosting services (like GitHub Pages) or browsers
give you a cached version of a page to save time, but sometimes it will be an old version that you don't want. In this case, we unfortunately don't have control over that. After some time (maybe a few hours, maybe a few days), a normal refresh would show you or any other visitor the latest page.

**Reply by rebullm** [reply]
Yes that must be it- I don't see it either right now. I'll try to be more patient in the future (!) Thanks!

**Reply by vincerubinetti** [reply]
A quick sidenote. I recommend doing larger PRs before merging into main. You can continue to make changes once you've opened a PR (try pressing `.` when on the PR page), and then saving/committing the changes will update the PR preview. GitHub Actions is free, but only up to a certain monthly quota. These runs do take up some of that, so it makes sense to minimize the amount of workflow runs, and doing a new PR then committing to main for every small change might start to add up.
